snuba/clickhouse/http.py

```diff
@@ -219,6 +220,14 @@ def join(self, timeout: Optional[float] = None) -> None:
                 code = int(details["code"])
                 message = details["message"]
                 row = int(details["row"]) if details["row"] is not None else None
```

Comment:
Could we start by simply including this `details["row"]` value that comes back from ClickHouse to the Sentry error instead of storing a copy of all the raw rows.

Reply:
edit: nvm, that is probably nothing useful. But what about attaching some of the other data that comes back from the ClickHouse error?

Reply:
@lynnagara I think the f/u from the incident was specifically about missing insight into the data that is being inserted, i don't think clickhouse returns that as part of the error. Is there something in particular that you would also like to have?

Reply:
Ah, ignore my comment then. I assumed we just wanted parts of the clickhouse error that was missing and might help us better diagnose. But that might not help us understand if bad data got there to begin with.